### ISSUE TITLE
Conversion of Pbutils Output Tensor to Numpy Array with or without dl pack

### ISSUE THREAD

**Issue by NavKumarGit** [triton-inference-server/server#4457]
**Description**
No direct way to convert the Pbutils Output Tensor to Numpy which is not on CPU. Using torch dl_pack adds 890MB in the GPU as python backend stub process

**Triton Information**
What version of Triton are you using?
22.04

Are you using the Triton container or did you build it yourself?
Triton Container with custom python code functions and installation of torch in it.
**To Reproduce**
Steps to reproduce the behavior.
call a TensorRT model inside of a python backend model.
To get the output from the received inference response following code is used:

request = pb_utils.InferenceRequest(model_name='model_a',
                                                        requested_output_names=["output"],
                                                        inputs=[A,B, C])

post_response = request.exec()

if post_response.has_error():
      raise pb_utils.TritonModelException(
                post_response.error().message())
else:        
         output = pb_utils.get_output_tensor_by_name(post_response, "output")
        logits = from_dlpack(output.to_dlpack())

running this logit using dlpack opens a Python backend stub in the GPU device in which the model_a is placed of ~900 MB
            

**Expected behavior**
It shouldn't open any stub of 900 MB, even if it opens it should open a stub of 97MB which I see it does for other python backend model if they call any tensor model from inside.

Or suggest any other way to convert the output tensor to the Numpy without using torch dl_pack package.
Can we have a Paramter: "FORCE_CPU_ONLY_OUTPUT_TENSORS" like it's there for INPUT Tensors.

**Comment by Tabrizian** [comment]
You can use any framework that supports DLPack to convert the GPU tensors. CuPy uses much less memory compared to PyTorch when initializing GPU tensors which may resolve your problem. Having said that, there is a feature request to add support for copying GPU tensors to CPU in Python backend.
